src/character.cpp

```diff
@@ -12433,6 +12442,17 @@ float Character::fall_damage_mod() const
     return std::max( 0.0f, ret );
 }
 
+static float adjust_effective_force_for_soft_landing( float effective_force )
+{
+    if( effective_force < 10.0 ) {
+        return 0.0f;  // If less than 10, reduce it to 0
+    } else if( effective_force < 25.0f ) {
+        return effective_force / 3.0f;  // If less than 25 but greater than or equal to 10, reduce by 3
+    } else {
+        return effective_force * 0.75f;  // Otherwise, reduce by 0.75
+    }
+}
+
```

Comment:
This should be a flat reduction to damage, probably something in the teens.

Reply:
Just realized if you make it a field it can have both options for multipliers and straight reductions.